solutions/observability/apm/use-opentelemetry-with-apm.md

```diff
@@ -46,65 +44,17 @@ With an Elastic Distribution of OpenTelemetry language SDK you have access to al
 
 Get started with an Elastic Distribution of OpenTelemetry language SDK:
 
-* [**Elastic Distribution of OpenTelemetry Java →**](https://github.com/elastic/elastic-otel-java)
-* [preview] [**Elastic Distribution of OpenTelemetry .NET →**](https://github.com/elastic/elastic-otel-dotnet)
-* [preview] [**Elastic Distribution of OpenTelemetry Node.js →**](https://github.com/elastic/elastic-otel-node)
-* [preview] [**Elastic Distribution of OpenTelemetry Python →**](https://github.com/elastic/elastic-otel-python)
-* [preview] [**Elastic Distribution of OpenTelemetry PHP →**](https://github.com/elastic/elastic-otel-php)
+* [**Elastic Distribution of OpenTelemetry Java →**](https://elastic.github.io/opentelemetry/edot-sdks/java/index.html)
+* [preview] [**Elastic Distribution of OpenTelemetry .NET →**](https://elastic.github.io/opentelemetry/edot-sdks/dotnet/index.html)
+* [preview] [**Elastic Distribution of OpenTelemetry Node.js →**](https://elastic.github.io/opentelemetry/edot-sdks/nodejs/index.html)
+* [preview] [**Elastic Distribution of OpenTelemetry Python →**](https://elastic.github.io/opentelemetry/edot-sdks/python/index.html)
+* [preview] [**Elastic Distribution of OpenTelemetry PHP →**](https://elastic.github.io/opentelemetry/edot-sdks/php/index.html)
```

Comment:
@AlexanderWert can we remove the `preview` for the individual EDOT SDKs? 

```suggestion
* [**Elastic Distribution of OpenTelemetry Java →**](https://elastic.github.io/opentelemetry/edot-sdks/java/index.html)
* [**Elastic Distribution of OpenTelemetry .NET →**](https://elastic.github.io/opentelemetry/edot-sdks/dotnet/index.html)
* [**Elastic Distribution of OpenTelemetry Node.js →**](https://elastic.github.io/opentelemetry/edot-sdks/nodejs/index.html)
* [**Elastic Distribution of OpenTelemetry Python →**](https://elastic.github.io/opentelemetry/edot-sdks/python/index.html)
* [**Elastic Distribution of OpenTelemetry PHP →**](https://elastic.github.io/opentelemetry/edot-sdks/php/index.html)
```

Reply:
I will do this in a follow up PR because there are other pages we should update.